go.mod

```diff
@@ -115,15 +115,15 @@ replace github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-
 // k8s deps pinning
 replace k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.16.4
 
-replace k8s.io/api => k8s.io/api v0.19.3
+replace k8s.io/api => k8s.io/api v0.22.0
```

Comment:
I'm ok with limiting it this way if needed, but why is it needed?

Reply:
without it, it will move towards newer version. i seeked for a smallest bump.

Reply:
sure, that's clear that without limiting the version, it will go with a higher version :)
my point was what blocked you from taking a higher version - but that's ok, we can bump them separately if there's no reason not to